scripts/automation/release/check.py

```diff
@@ -0,0 +1,43 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=line-too-long
+from __future__ import print_function
+
+import argparse
+from subprocess import check_call
+
+from ..utilities.path import get_repo_root, get_all_module_paths
+
+def check_component_revisions(component_name, r_start, r_end):
+    for comp_name, comp_path in get_all_module_paths():
+        if comp_name == component_name:
+            revision_range = "{}..{}".format(r_start, r_end)
+            check_call(["git", "log", "--pretty=format:'%C(yellow)%h %Cred%ad %Cblue%an%Cgreen%d %Creset%s'", revision_range, "--", comp_path], cwd=get_repo_root())
```

Comment:
But would it show the changes to both or just the changes to the requested module?